web/styles/app_variables.css

```diff
@@ -792,6 +796,10 @@
     --color-border-emoji-picker-emoji-reacted: hsl(0deg 0% 0% / 90%);
     --color-background-emoji-picker-emoji-reacted-focus: hsl(0deg 0% 20% / 70%);
     --color-border-add-subscription-button-focus: hsl(0deg 0% 67%);
+
+    /* Rendered markdown table color (dark) */
```

Comment:
Again with the reorganization. Be sure you order the new variables the same as in the light area of the file, too. These are flipped.